templates/pageLevelProgressItem.hbs

```diff
@@ -0,0 +1,13 @@
+<button
+    class="base pagelevelprogress-item-title clearfix drawer-item-open{{#unless _isVisible}} disabled{{/unless}} js-indicator"
+    data-pagelevelprogress-id="{{_id}}"
+    aria-label="{{#unless _isVisible}}{{_globals._accessibility._ariaLabels.locked}}. {{/unless}}{{#if _isOptional}}{{_globals._extensions._pageLevelProgress.optionalContent}}{{else}}{{#if _isComplete}}{{_globals._accessibility._ariaLabels.complete}}{{else}}{{_globals._accessibility._ariaLabels.incomplete}}{{/if}}{{/if}} {{compile title}}">
```

Comment:
probably need to add full stop after complete/incomplete aria label; currently it will display something like "Locked. Incomplete Component title"

Reply:
Might it not read better with the order reversed i.e. "Component title. Locked. Incomplete."?

Reply:
Not if you're trying to get through at speed. "Complete" things can generally be ignored. If one has to wait for the title to read first, it takes aaaaages to figure out what to navigate to.

Reply:
still think it could use a full stop though...